modules/test/playwright/pages/commerce/commerce-product-content-web/productPublisherPage.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {FrameLocator, Locator, Page} from '@playwright/test';
+
+import {CommerceLayoutsPage} from '../commerceLayoutsPage';
+
+export class ProductPublisherPage {
+	readonly closeButton: Locator;
+	readonly configurationFilterLink: Locator;
+	readonly configurationFrame: FrameLocator;
+	readonly configurationMenuItem: Locator;
+	readonly configurationOrderingLink: Locator;
+	readonly configurationSaveButton: Locator;
+	readonly layoutsPage: CommerceLayoutsPage;
+	readonly optionsButton: Locator;
+	readonly page: Page;
+	readonly productLink: (productName: string) => Promise<Locator>;
+	readonly removeTagNameButton: (tagName: string) => Promise<Locator>;
+	readonly tagsInput: Locator;
+
+	constructor(page: Page) {
+		this.closeButton = page.getByLabel('close', {exact: true});
+		this.configurationFilterLink = page
+			.frameLocator(
+				'iframe[title*="Product Publisher"][title*="Configuration"]'
+			)
+			.getByRole('link', {exact: true, name: 'Filter'});
```

Comment:
nice to have: this should be 
```
this.configurationFrame.getByRole...
```
but you have to save it after the other variable

Reply:
I did like this to maintain the alphabetical order